kubernetes/cmsweb/scripts/deploy-secrets.sh

```diff
@@ -1,24 +1,31 @@
 #!/bin/bash
 # helper script to deploy given service with given tag to k8s infrastructure
 
-if [ $# -ne 3 ]; then
-    echo "Usage: deploy-secrets.sh <namespace> <service-name> <path_to_configuration>"
+if [ $# -ne 4 ]; then
+    echo "Usage: deploy-secrets.sh <namespace> <service-name> <path_to_configuration> <reference_of_decryption_barbican_secret>"
```

Comment:
@Panos512 

This secret reference is quite long string. i.e., ``https://openstack.cern.ch:9311/v1/secrets/35145b52-18af-47a3-90d0-1xxxxxxxxxxxxxx` users should not remember this string. Instead based on the `env`, we can use  existing key in this script?